index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
+          <ul class="company-nav">
+            <li class="company-item">
+              <h3 class="company-heading">Company</h3>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Homepage v1</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Homepage v2</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Homepage v3</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">About Us</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Creators</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Blog Articles</a>
+            </li>
+          </ul>
+          <ul class="products-nav">
+            <li class="products-item">
+              <h3 class="products-heading">Products</h3>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Website</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Mobile App</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Social Media Feeds</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Presentation</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Product Mockups</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Illustrations</a>
+            </li>
+          </ul>
+          <ul class="account-nav">
+            <li class="account-item">
+              <h3 class="account-heading">Account</h3>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Your Cart</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Check Out</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Refund Policy</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Company Policy</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Send A Message</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Customer Support</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="footer-text-container">
+          <img src="#" alt="#" class="footer-logo" />
+          <p class="footer-text">
+            Your premium digital assets solution with high quality and
+            comprehensive range of products and services.
+          </p>
+          <div class="social-links">
```

Comment:
Homogeneous repeating elements (navigation; tags perceived as a list) should be implemented as a list - http://htmlbook.ru/html/ul